drivers/display/ssd1306.py

```diff
@@ -37,12 +37,12 @@ def __init__(self, width, height, external_vcc):
 
     def init_display(self):
         for cmd in (
-            SET_DISP | 0x00,  # off
+            SET_DISP,  # display off
```

Comment:
What was the reason to remove the `| 0x00`?  The compiler *should* be optimising this away because `SET_DISP` is a constant (that does require constant folding to be enabled, which it is for most ports).

Reply:
Just seemed a bit redundant and worked fine without it on my ESP32. I can pop it back in if needed for other ports

Reply:
It's not needed, and will be slightly more efficient without it on all ports (because the compiler doesn't have to spend time optimising it).  So let's go with the change!